src/WebJobs.Script.WebHost/Diagnostics/ILoggingBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Logging
+{
+    public static class ILoggingBuilderExtensions
+    {
+        public static void AddWebJobsSystem<T>(this ILoggingBuilder builder) where T : SystemLoggerProvider
+        {
+            builder.Services.AddSingleton<ILoggerProvider, T>();
+
+            // Log all debug-level logs to SystemLogger
```

Comment:
is this comment not correct?

Reply:
Whoops. Yeah, I originally had it set at Debug but noticed that we log some useful stuff at Trace level. I'll adjust the comment.